sky/data/storage.py

```diff
@@ -4004,25 +3945,23 @@ def _download_file(self, remote_path: str, local_path: str) -> None:
           local_path: str; Local path on user's device
         """
         if remote_path.startswith(f'/{self.name}'):
-            _, oos_path = data_utils.split_oci_path(remote_path)
-            remote_path = oos_path
+            # If the remote path is /bucket_name, we need to
+            # remove the leading /
+            remote_path = remote_path.lstrip()
```

Comment:
```suggestion
            remote_path = remote_path.lstrip('/')
```
should this removing `/` ?

Reply:
Oh yes, thanks a lot :)